src/solvers/dg_unstructured_quad/dg_2d.jl

```diff
@@ -7,10 +7,6 @@ function create_cache(mesh::UnstructuredQuadMesh, equations,
   polydeg_ = polydeg(dg.basis)
   nvars = nvariables(equations)
 
-  if polydeg_ > mesh.polydeg && any(mesh.element_is_curved)
-    throw(ArgumentError("polynomial degree of DG (= $polydeg_) must be less than or equal to mesh polynomial degree (= $(mesh.polydeg))"))
-  end
```

Comment:
This check is basically replaced by the one below, isn't it?

Reply:
yes, I like the new check because it a "stronger" test is my eyes because it ensures that your approximation will be free-stream preserving even when you do not explicitly run that elixir whereas the check above would allow this to happen. That is, the check above hid a bug from me where the scheme converged but was not free-stream

Reply:
Can you still run `Trixi$ julia benchmark/elixir_2d_euler_vortex_unstructured.jl` without problems? If so, feel free to resolve this thread.